src/Microsoft.Data.SqlClient/tests/ManualTests/AlwaysEncrypted/TestFixtures/SQLSetupStrategy.cs

```diff
@@ -89,6 +90,14 @@ internal virtual void SetupDatabase()
                 }
 
                 // Insert data for TrustedMasterKeyPaths tests.
+                InsertSampleData(TrustedMasterKeyPathsTestTable.Name);
+            }
```

Comment:
```suggestion
            }
            // Insert data for TrustedMasterKeyPaths tests.
            InsertSampleData(TrustedMasterKeyPathsTestTable.Name);
```

If `AEConnStringsSetup` has multiple values, I think this could be the reason why tests are failing. We create the table in the first data source, then call `InsertSampleData` - but this iterates over every data source, so when it reaches the second it's trying to address a table which doesn't yet exist.

I think there might be another test failure to come, but this should at least let us see it...